src/containers/HomePage.tsx

```diff
@@ -53,89 +53,150 @@ const HomePage: React.FunctionComponent = () => {
           </div>
-        </div>
-        <div className="section section-5 container">
-          <div className="sub-section">
-            <img className="small-image" alt="home-espot-4" src={Config.siteImagesUrl.replace('%fileName%', homeEspot4FileName)} onError={(e: any) => { e.target.src = homeEspot4; }} />
-            <div className="text-block">
-              <span className="line line-1">
-                {intl.get('home-sub-espot-container4-first-line')}
-              </span>
-              <span className="line line-2">
-                {intl.get('home-sub-espot-container4-second-line')}
-              </span>
+          <div className="section section-5 container">
+            <div className="sub-section">
+              <img className="small-image" alt="home-espot-4" src={Config.siteImagesUrl.replace('%fileName%', homeEspot4FileName)} onError={(e: any) => { e.target.src = homeEspot4; }} />
+              <div className="text-block">
+                <span className="line line-1">
+                  {intl.get('home-sub-espot-container4-first-line')}
+                </span>
+                <span className="line line-2">
+                  {intl.get('home-sub-espot-container4-second-line')}
+                </span>
+              </div>
             </div>
           </div>
-        </div>
-        <div className="section-parallax section-parallax-2 container" data-region="homeMainContentRegion">
-          <Parallax y={[10, -30]} tagOuter="figure">
-            <div className="parallax-image-container">
-              <img className="parallax-image" alt="home-espot-1" src={Config.siteImagesUrl.replace('%fileName%', homeEspotParallax2FileName)} onError={(e: any) => { e.target.src = homeEspotParallax2; }} />
-            </div>
-          </Parallax>
-          <div className="sub-section">
-            <div className="text-block">
-              <span className="line line-white">
-                {intl.get('home-sub-espot-container2-second-line')}
-              </span>
-              <span className="line line-link">
+          <div className="section-parallax section-parallax-2 container" data-region="homeMainContentRegion">
+            <Parallax y={[10, -30]} tagOuter="figure">
+              <div className="parallax-image-container">
+                <img className="parallax-image" alt="home-espot-1" src={Config.siteImagesUrl.replace('%fileName%', homeEspotParallax2FileName)} onError={(e: any) => { e.target.src = homeEspotParallax2; }} />
+              </div>
+            </Parallax>
+            <div className="sub-section">
+              <div className="text-block">
+                <span className="line line-white">
+                  {intl.get('home-sub-espot-container2-second-line')}
+                </span>
+                <span className="line line-link">
                 Learn more
```

Comment:
I know this has been here already, but can we please replace it with the localized string version? We shouldn't have hardcoded strings